Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateMediaSourceAVFObjC.mm

```diff
@@ -576,7 +576,7 @@ void getSupportedTypes(HashSet<String>& types) const final
 {
     if (m_seekState == SeekCompleted)
         return;
-    if (hasVideo() && !m_hasAvailableVideoFrame) {
+    if (layerOrVideoRenderer() && !m_hasAvailableVideoFrame) {
```

Comment:
It isn't new with this PR, but it would be nice to rename `layerOrVideoRenderer` to `hasLayerOrVideoRenderer` or something that better describes what the method does.